packages/material-ui/src/StepConnector/StepConnector.js

```diff
@@ -87,6 +117,9 @@ StepConnector.propTypes = {
 StepConnector.defaultProps = {
   alternativeLabel: false,
   orientation: 'horizontal',
+  active: false,
```

Comment:
Do we need the default value if they are not documented?

Reply:
You are right, we don't need to define default values. The default value is `undefined` anyway so `[classes.lineActive]: active` will work as expected.